robots/EV3D4/EV3D4RemoteControl.py

```diff
@@ -2,8 +2,9 @@
 
 import logging
 import sys
-from ev3dev.auto import OUTPUT_A, OUTPUT_B, OUTPUT_C
-from ev3dev.helper import RemoteControlledTank, MediumMotor
+from ev3dev.motor import OUTPUT_A, OUTPUT_B, OUTPUT_C, MediumMotor
```

Comment:
Is this the import style we're recommending now (specifying all of the types to import)? I think it might be better to recommend importing `*`in the interest of easy set-up guides for beginners.

Reply:
Importing `*` is generally viewed as bad practice because it makes it difficult to determine what functions/variables came from where.

I try to only import what we need just to make the load times a little faster.

Reply:
The convention makes sense, but I'm finding it reasonably difficult to explain succinctly how to manage import statements if every time you copy an example you need to figure out what new classes are needed. I doubt that most beginning programmers will have any idea what the error means when they use a class that isn't imported.

Reply:
Maybe we could do something where in the demos if we are importing motor we import
```
from ev3dev.motor import OUTPUT_A, OUTPUT_B, OUTPUT_C, OUTPUT_D, LargeMotor, MediumMotor
```
even if we aren't using all of those, just to point the user in the right direction?